docs/elements.md

```diff
@@ -89,6 +88,8 @@ Note: PrairieLearn Elements listed next have been
 **deprecated**. These elements are still supported for backwards
 compatibility, but they should not be used in new questions.
 
+- [`pl-dropdown`](#pl-dropdown-element): Select an answer from answers in a drop-down box.
+  - **Deprecated**: use [`pl-multiple-choice`](#pl-multiple-choice-element) or [`pl-matching`](#pl-matching-element) instead.
```

Comment:
Could we give a bit more guidance here on when to use each of these alternatives, and which attributes to set with them to get dropdown-like behavior? Basically a copy of the warning text in the `pl-dropdown` section.